### DISCUSSION TITLE
Just like tax, have keyword for indirect taxes too, like gst.

### DISCUSSION THREAD

**Discussion by avinashkanaujiya** [ananthakumaran/paisa#51] [Ideas]
Just something came to me while entering my expenses, maybe we can also track our indirect tax outgo. just like regular tax, if we just a had keyword - "gst", just like we have for tax.

**Comment by ananthakumaran** [comment]
Have you tried using sub accounts, like `Expense:Tax:Income`, `Expense:Tax:Gst` etc? It would still get treated as a tax account.

**Reply by avinashkanaujiya** [reply]
Good Idea. Didn't think.

**Reply by avinashkanaujiya** [reply]
Doesn't work, when I updated salary with Expenses:Tax:Salary, it stopped showing up in Tax section in Expenses page, and got counted in expenses and tax went to zero.

**Reply by ananthakumaran** [reply]
Ah, I was thinking about the checking account. fixed on master https://github.com/ananthakumaran/paisa/commit/96bc28c
